### PR TITLE
Publish artifacts for semantic-convention-utils module

### PR DESCRIPTION
**Pull request by rish691**

### CODE REVIEW

File: semantic-convention-utils/build.gradle.kts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 plugins {
     `java-library`
+    id("org.hypertrace.publish-plugin")
 }
 
+group = "org.hypertrace.utils"
```
Comment (kotharironak) [inline semantic-convention-utils/build.gradle.kts:6]:
```suggestion
group = "org.hypertrace.semanticconventionutils"
```

Comment (kotharironak) [reply semantic-convention-utils/build.gradle.kts:6]: or `semconutils` ?

Comment (kotharironak) [reply semantic-convention-utils/build.gradle.kts:6]: This `org.hypertrace.utils` is very generic. Pl. add appropriate prefix.

Comment (rish691) [reply semantic-convention-utils/build.gradle.kts:6]: `org.hypertrace.utils` this is the group id right? 
Consumers can use it like: `org.hypertrace.utils:semantic-convention-utils:<version>`

**Review by kotharironak — APPROVED** [review]

**Pull request merged**